app/queries/get_linkables.rb

```diff
@@ -33,7 +33,7 @@ def call
           state_fallback_order: [:published, :draft]
         )
 
-        Edition.where(id: edition_ids).map do |edition|
+        Edition.with_document.includes(:document).where(id: edition_ids).map do |edition|
```

Comment:
I think we use it indirectly because we access document fields with the presenter.

My understanding is without the join, activerecord will run two queries to get the data to populate the editions with eager loaded documents. One for the Edition records, and one for the Document records. Whereas with the join it will get all the data in a single query.